Sources/AloeStackView/AloeStackView.swift

```diff
@@ -64,35 +64,39 @@ open class AloeStackView: UIScrollView {
   /// Adds a row to the end of the stack view.
   ///
   /// If `animated` is `true`, the insertion is animated.
-  open func addRow(_ row: UIView, animated: Bool = false) {
-    insertCell(withContentView: row, atIndex: stackView.arrangedSubviews.count, animated: animated)
+  open func addRow(_ row: UIView, animated: Bool = false, completion: ((Bool) -> Void)? = nil) {
+    insertCell(withContentView: row, atIndex: stackView.arrangedSubviews.count, animated: animated, completion: completion)
   }
 
   /// Adds multiple rows to the end of the stack view.
   ///
   /// If `animated` is `true`, the insertions are animated.
-  open func addRows(_ rows: [UIView], animated: Bool = false) {
-    rows.forEach { addRow($0, animated: animated) }
+  open func addRows(_ rows: [UIView], animated: Bool = false, completion: ((Bool) -> Void)? = nil) {
+    let group = DispatchGroup()
+    rows.forEach { group.enter(); addRow($0, animated: animated) { _ in group.leave() } }
+    group.notify(queue: .main) { completion?(true) }
   }
 
   /// Adds a row to the beginning of the stack view.
   ///
   /// If `animated` is `true`, the insertion is animated.
-  open func prependRow(_ row: UIView, animated: Bool = false) {
-    insertCell(withContentView: row, atIndex: 0, animated: animated)
+  open func prependRow(_ row: UIView, animated: Bool = false, completion: ((Bool) -> Void)? = nil) {
+    insertCell(withContentView: row, atIndex: 0, animated: animated, completion: completion)
   }
 
   /// Adds multiple rows to the beginning of the stack view.
   ///
   /// If `animated` is `true`, the insertions are animated.
-  open func prependRows(_ rows: [UIView], animated: Bool = false) {
-    rows.reversed().forEach { prependRow($0, animated: animated) }
+  open func prependRows(_ rows: [UIView], animated: Bool = false, completion: ((Bool) -> Void)? = nil) {
```

Comment:
please wrap these function signatures on to multiple lines
```
open func prependRows(
    _ rows: [UIView],
    animated: Bool = false,
    completion: ((Bool) -> Void)? = nil)
{
```

Reply:
i broke the line of function name that longer then 110 characters.

Reply:
however, here is my break line style.

```
open func prependRows(_ rows: [UIView],
                      animated: Bool = false,
                      completion: ((Bool) -> Void)? = nil) {
       // code is here
}
```

if you don't like this style please commit the additional feedback. i will apply as soon as possible.